sdl2promela/sdl2promela.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import typing
+from typing import List
+import logging
+import opengeode
+import sys
+import traceback
+
+from opengeode import ogAST
+from .sdl import model as sdlmodel
+from .promela import model as promelamodel
+from .promela import generator as promelagenerator
+from . import translator
+from . import __version__
+
+log = logging.getLogger("sdl2promela")
+
+def read_process(sdl_files : List[str]) -> ogAST.Process:
+    try:
+        ast, warnings, errors = opengeode.parse(sdl_files)
+    except IOError as error:
+        log.error("IOError:" + error)
+        traceback.print_exc()
+        sys.exit(1)
+    except:
+        log.error("Unknown parsing error")
+        traceback.print_exc()
+        sys.exit(1)
+
+    if len(warnings) > 0:
+        log.error(f"Error: found {len(errors)} warnings (warnings indicate issues with the SDL model and are treated as errors):")
+        for warning in warnings:
+            log.error(warning)
+        sys.exit(1)
```

Comment:
Exit due to warnings in SDL code? I'm not sure, but I think that this might be good approach.

Reply:
Yes. If this will cause issues, I will reconsider. However, a well formed SDL model should not have any warnings, and I assume that processing a malformed one may fail in best case, or taint the results in worst case.